Sources/FluentMongoDriver/GridFS.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import MongoKitten
+import FluentKit
+
+extension MongoDatabaseRepresentable {
+    internal var _gridFS: GridFSBucket {
```

Comment:
If it's already internal, why have the `_` here?

Reply:
I wanted to expose a GridFSBucket wrapper at `gridFS` at first, but then I decided to move those functions to GridFSFile